mentorApp/lib/FTU/SignUp.dart

```diff
@@ -34,9 +36,9 @@ class SecondRoute extends StatelessWidget {
         child: RaisedButton(
           onPressed: () {
             // Navigate back to first route when tapped.
-            Navigator.pushNamed(context, '/UserProfile');
```

Comment:
I'll have to test this out tomorrow, but I believe you could just call `UserProfile()` here to load the screen.